cwltool/builder.py

```diff
@@ -301,13 +303,15 @@ def bind_input(self,
                         if isinstance(sf, dict) or "$(" in sf or "${" in sf:
                             sfpath = self.do_eval(sf, context=datum)
                         else:
-                            if sf.endswith('?'):
+                            if sf.endswith('?') and \
+                                            self.cwl_version == 'v1.1.0-dev1':
```

Comment:
@tetron @mr-c  I added this naive check for the CWL version for now. I guess this should be done in a nicer way, to cover all the versions of the future and the past?